<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGeneralTuning</class>
 <widget class="QMainWindow" name="FormGeneralTuning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Tuning</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>380</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
 	background-color: rgb(255, 255, 255);
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtKP">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtKI">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>KI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtI2VGain">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>I-to-V Gain</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOffsetError">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset Error</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMindIdt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Iref</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMinIref">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGainError">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxIref">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum dIdt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Iref</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain Error</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterlockError">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock Voltage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max dIdt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterlockIload">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterlockIEartth">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock Error</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterlockVoltage">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxTotalE">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock dIdt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxdIdt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock ILoad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock IEarth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max E Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min E Total</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMinTotalE">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterlockdIdt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterlockTemp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock Temperature</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interlock Mask</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterlockMask">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
